tests/devices_test/device_properties_test.py

```diff
@@ -759,6 +762,8 @@ def __init__(self, methodName="run_test"):
             "parents": xform(lambda x, m: self.assertEqual(len(x), 1, m) and
                              self.assertIsInstance(x, ParentList) and
                              self.assertIsInstance(x[0], LVMVolumeGroupDevice)),
+            "size": xform(lambda x, m: self.assertEqual(x, Size("2 MiB"), m)),
+            "target_size": xform(lambda x, m: self.assertEqual(x, Size("2 MiB"), m))
```

Comment:
Would be better if instead of `Size("2 MiB")` you used `get_device_format_class("fs")._min_size` or similar.

Reply:
Fixed.